pandas/_libs/parsers.pyx

```diff
@@ -558,13 +564,7 @@ cdef class TextReader:
         pass
 
     def __dealloc__(self):
-        parser_free(self.parser)
-        if self.true_set:
```

Comment:
are these de-allocations handledede elsewhere?

Reply:
`close()` has the same code